src/components/ButtonGroup.tsx

```diff
@@ -0,0 +1,53 @@
+import { SortType } from '../enums/sort-type.enum';
+import { Button } from './Button';
+
+type Props = {
+  sortField: SortType | '';
+  isReversed: boolean;
+  setSortField: (sortField: SortType | '') => void;
+  setIsReversed: (isReversed: boolean) => void;
+  reset: () => void;
+};
+
+export const ButtonGroup: React.FC<Props> = ({
+  sortField,
+  isReversed,
+  setSortField,
+  setIsReversed,
+  reset,
+}) => {
+  const handleSortAlphabetically = () => setSortField(SortType.ALPHABETICALLY);
+  const handleSortByLength = () => setSortField(SortType.BY_LENGTH);
+  const handleReverse = () => setIsReversed(!isReversed);
+  const handleReset = () => reset();
```

Comment:
[CODE STYLE] - Function's names should start with a verb to clearly indicate what the method does. Consider renaming `handleSortAlphabetically` to `sortAlphabetically`, `handleSortByLength` to `sortByLength`, `handleReverse` to `toggleReverse`, and `handleReset` to `resetFields` or similar.

Reply:
ok